docs/book/src/SUMMARY.md

```diff
@@ -2,19 +2,21 @@
 
 [Introduction](./introduction.md)
 
-[Quick Start](./quick-start.md)
+- [User Guide](./user/guide.md)
+    - [Concepts](./user/concepts.md)
+    - [Quick Start](./user/quick-start.md)
 
-- [Architecture](./architecture/architecture.md)
+- [Developer Guide](./architecture/architecture.md)
```

Comment:
Instead of this linking to architecture, it should probably link to a developer guide main page.

Reply:
Fixed!